loadtests/Makefile

```diff
@@ -12,11 +12,12 @@ INSTALL = ARCHFLAGS=$(ARCHFLAGS) ./venv/bin/pip install
 ./venv/.env.install:
 	virtualenv ./venv
 	$(INSTALL) pexpect >> loadtest.log && \
-	$(INSTALL) configparser==3.3.0post2 >> loadtest.log && \
+	$(INSTALL) configparser==3.5.0b2 >> loadtest.log && \
```

Comment:
This actually depends on the pip version you are using. Last version of pip won't find the 3.5.0b2, lastly travis couldn't either.